### PR TITLE
[RU] Add 'basics/strings' lesson translation

### PR DESCRIPTION
**Pull request by nscyclone**
@brain-geek, could you, please, take a peek at this one?

### CODE REVIEW

File: ru/lessons/basics/strings.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
+---
+layout: page
+title: Строки
+category: basics
+order: 14
+lang: ru
+---
+
+Строки, списки символов, графемы и кодовые точки.
```
Comment (brain-geek) [inline ru/lessons/basics/strings.md]: I guess 'графемы' should be added to glossary - I haven't heard this term.

Comment (brain-geek) [inline ru/lessons/basics/strings.md]: Same about `кодовые точки` - I think that's `codepoint`?

File: ru/lessons/basics/strings.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+```elixir
+iex> string = <<104,101,108,108,111>>
+"hello"
+```
+
+>ПРИМЕЧАНИЕ: Используя << >>, мы сообщаем компилятору, что элементы внутри этих символов &mdash; это байты.
+
+## Списки символов
+
+Внутри Elixir строки представлены в виде последовательности байтов, а не списка символов. Elixir также поддерживает и этот тип (списки символов). Строки в Elixir заключаются в двойные кавычки, а списки символов &mdash; в одинарные.
```
Comment (brain-geek) [inline ru/lessons/basics/strings.md]: What do you think about changing first two sentenses to `В языке Elixir есть списки символов, однако строки представлены в виде последовательности байтов.`?

File: ru/lessons/basics/strings.md [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+'hello'
+
+iex> {hd, tl}
+{104, 'ello'}
+
+iex> Enum.reduce(char_list, "", fn char, acc -> acc <> to_string(char) <> "," end)
+"104,101,108,108,111,"
+```
+
+Программируя на Elixir, мы обычно используем строки, а не списки символов. Поддержка списков символов включена, в основном, потому что они необходимы для некоторых модулей Erlang.
```
Comment (brain-geek) [inline ru/lessons/basics/strings.md]: `потому что они необходимы для некоторых модулей` -> `для совместимости с некоторыми модулями`

File: ru/lessons/basics/strings.md [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+iex> String.codepoints string
+["a", "́"]
+
+iex> String.graphemes string
+["á"]
+```
+
+## Строковые функции
+
+Рассмотрим некоторые самые важные и полезные функции модуля строки. Этот урок покрывает лишь часть доступных функций; чтобы ознакомиться с полным списком, можно посетить [страницу](http://elixir-lang.org/docs/stable/elixir/String.html) официальной документации.
```
Comment (brain-geek) [inline ru/lessons/basics/strings.md]: I guess `;` can be changed to `.` here

File: ru/lessons/basics/strings.md [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+Возвращает количество графем в строке.
+
+```elixir
+iex> String.length "Hello"
+5
+```
+
+### `replace/3`
+
+Возвращает новую строку, заменив в оригинальной выбранное выражение на некоторую строку замены.
```
Comment (brain-geek) [inline ru/lessons/basics/strings.md]: `оригинальной` -> `исходной строке`

File: ru/lessons/basics/strings.md [hunk 6]
```diff
@@ -0,0 +1,151 @@
...
+  def sort_string(string) do
+    string
+    |> String.downcase
+    |> String.graphemes
+    |> Enum.sort
+  end
+end
+```
+
+Давайте взглянем на `anagrams?/2`. Мы проверяем, являются ли параметры бинарными. Именно таким образом в Elixir проверяют, является ли параметр строкой.
```
Comment (brain-geek) [inline ru/lessons/basics/strings.md]: `являются ли параметры бинарными` this made my reading stop. I think it should be rephrased, but no suggestions yet.

File: ru/lessons/basics/strings.md [hunk 7]
```diff
@@ -0,0 +1,151 @@
...
+
+После этого мы просто вызываем функцию, сортирующую строку в алфавитном порядке, предварительно переведя строки в нижний регистр и использовав `String.graphemes`, чтобы получить список графем в строке. Довольно очевидно, не так ли?
```
Comment (brain-geek) [inline ru/lessons/basics/strings.md]: `После этого` -> `Дальше`

**Comment by nscyclone** [comment]
Thanks for the review, @brain-geek! I've just updated everything according to it. 
Concerning the line 133 I don't see any better solution besides using `бинарные данные`. 
Could you check the whole translation once more, please?

File: ru/lessons/basics/strings.md [hunk 8]
```diff
@@ -0,0 +1,151 @@
...
+  def sort_string(string) do
+    string
+    |> String.downcase
+    |> String.graphemes
+    |> Enum.sort
+  end
+end
+```
+
+Давайте взглянем на `anagrams?/2`. Мы проверяем, являются ли параметры бинарными данными. Именно таким образом в Elixir проверяют, является ли параметр строкой.
```
Comment (brain-geek) [inline ru/lessons/basics/strings.md]: `являются ли параметры` -> `является ли каждый из параметров`
`проверяют, является ли параметр строкой` -> `можно проверить, что параметр является строкой`

File: ru/lessons/basics/strings.md [hunk 9]
```diff
@@ -0,0 +1,151 @@
...
+
+iex> Anagram.anagrams?("María", "íMara")
+true
+
+iex> Anagram.anagrams?(3, 5)
+** (FunctionClauseError) no function clause matching in Anagram.anagrams?/2
+    iex:2: Anagram.anagrams?(3, 5)
+```
+
+Как вы могли заметить, последний вызов `anagrams?` завершился с FunctionClauseError. Эта ошибка говорит нам о том, что в нашем модуле нет функции, которая могла бы принимать два небинарных аргумента, и это именно то, что мы хотим &mdash; просто принимать две строки и ничего больше.
```
Comment (brain-geek) [inline ru/lessons/basics/strings.md]: `, и` -> `. И`

I think that making this a new sentence would be nice idea.

**Comment by brain-geek** [comment]
@nscyclone I've looked through the translation once again and found few more spaces to improve. And a suggestion to make 133 sound more natural.

**Comment by nscyclone** [comment]
Thanks again, @brain-geek! Could you take another peek at this just to ensure that we didn't miss anything important?

**Comment by brain-geek** [comment]
@nscyclone I still feel that something is wrong with the `бинарными данными`, but no suggestions there from me.

Otherwise, it's awesome. Thanks!

**Pull request merged**